lib/src/header_parser/utils.dart

```diff
@@ -197,8 +197,9 @@ String? removeRawCommentMarkups(String? string) {
   string = string.replaceAll('/*', '');
   string = string.replaceAll('*/', '');
 
-  // Remove any *'s in the beginning of a every line.
+  // Remove any *'s or // in the beginning of a every line.
```

Comment:
nit: it might someday be worth doing a bit of a heuristic here.

Like:
 * If comment starts with `\s*/\**?` then:
    * remove `\s*/\**?` from the start of the comment,
    * remove `*/\s*` at the end of the comment,
    * if every line starts with `\s\*` then remove `\s\*` from start of every line.
 * If comment starts with `\s*///?` then remove `\s*///?` from start of every line.

Reply:
Done.